shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/Linker.kt

```diff
@@ -123,7 +124,9 @@ class AndroidLinker(targetProperties: AndroidConfigurables)
                                    libraries: List<String>, linkerArgs: List<String>,
                                    optimize: Boolean, debug: Boolean,
                                    kind: LinkerOutputKind, outputDsymBundle: String,
-                                   needsProfileLibrary: Boolean, mimallocEnabled: Boolean): List<Command> {
+                                   needsProfileLibrary: Boolean, mimallocEnabled: Boolean,
+                                   sanitizer: SanitizerKind): List<Command> {
+        if (sanitizer != SanitizerKind.NONE) throw Error("Sanitizers are unsupported")
```

Comment:
(minor) It's possible to use standard functions instead of plain `if`s: [`check`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/check.html), [`require`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html)